packages/dialect-react-ui/components/Notifications/Notification.tsx

```diff
@@ -17,15 +17,15 @@ const timeFormatter = new Intl.DateTimeFormat('en-US', {
 });
 
 export const Notification = ({ message, timestamp }: Props) => {
-  const { textStyles } = useTheme();
+  const { textStyles, notificationBubble, notificationTimestamp } = useTheme();
```

Comment:
`notificationBubble` is the individual notification message, correct? What do you think about `notificationMessage` as the name?

Reply:
yeah, good suggestion, renamed